src/app/dashboard/dashboard.component.ts

```diff
@@ -139,7 +139,7 @@ export class DashboardComponent implements OnInit, OnDestroy {
         this.getData('teams', memberships.map((doc: any) => doc.teamId), { titleField: 'name', linkPrefix: '/teams/view/', addId: true })
       ])),
       map(([ memberships, teams ]: any[]) =>
-        teams.filter(team => team.type === undefined || team.type === 'team').map(team => ({
+        teams.map(team => ({

```

Comment:
there are teams without type as well. These are usually members and there are also records like credit. So removing filter might display other records as well.

Reply:
Reinstated the filter and additionally filtering for type === enterprises